### PR TITLE
feat: Backup wallet screen - intro

### PR DESCRIPTION
**Pull request by andrzejchm**

### CODE REVIEW

File: starport_template/lib/pages/create_wallet_page.dart [hunk 1]
```diff
@@ -8,12 +15,100 @@ class CreateWalletPage extends StatefulWidget {
...
+            SizedBox(height: CosmosTheme.of(context).spacingL),
+            const InfoCard(text: "Never share your recovery phrase with anyone, store it securely."),
+            SizedBox(height: CosmosTheme.of(context).spacingL),
+            const InfoCard(
+              text: "If you don’t backup your wallet or lose your recovery phrase, "
+                  "you will not able to recover your account",
+            ),
+            const Spacer(),
+            CosmosElevatedButton(text: "Back up now", onTap: _onTapBackUpNow),
+            const SizedBox(height: 10),
```
Comment (wal33d006) [inline starport_template/lib/pages/create_wallet_page.dart:88]: Can we use spacing from `CosmosTheme.of(context)` here?

Comment (andrzejchm) [reply starport_template/lib/pages/create_wallet_page.dart:88]: yup! absolutely

**Review by wal33d006 — APPROVED** [review]
Just one comment, and good to go!

File: starport_template/lib/pages/onboarding_page.dart [hunk 2]
```diff
@@ -19,45 +19,48 @@ class OnboardingPage extends StatefulWidget {
...
+                    Text(
+                      "Access your wallet for alicechain/1.",
+                      style: Theme.of(context).textTheme.caption,
+                    ),
+                    const SizedBox(height: 38),
+                    CosmosElevatedButton(
+                      text: "Create account",
+                      onTap: _onTapCreateAccount,
+                    ),
+                    const SizedBox(height: 10),
```
Comment (wal33d006) [inline starport_template/lib/pages/onboarding_page.dart:53]: Same for these. Spacings from `CosmosTheme.of(context)`

**Pull request merged**